taipy/core/_scheduler/_scheduler_factory.py

```diff
@@ -9,17 +9,31 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+from importlib import util
+
 from taipy.core._scheduler._abstract_scheduler import _AbstractScheduler
 from taipy.core._scheduler._scheduler import _Scheduler
 from taipy.core.common._utils import _load_fct
 from taipy.core.config.config import Config
 
 
 class _SchedulerFactory:
+
     @classmethod
     def _build_scheduler(cls) -> _AbstractScheduler:
+
         if Config.job_config._is_default_mode():
-            return _Scheduler()
+            if util.find_spec("taipy.enterprise"):
+                # Mode standalone with enterprise version
+                package = "taipy.enterprise.core.scheduler"
```

Comment:
Can we do:
```suggestion
                package = "taipy.enterprise.scheduler"
```
?
In this case, we can do what we did with airflow.

Reply:
I believe `core` is needed from the PR Jean-Robin proposed earlier (https://github.com/Avaiga/taipy-enterprise/pull/3/files)
I actually proposed to have "taipy.enterprise.core.scheduler.scheduler" to be consistent with the folder architecture from taipy core. What do you think?

Reply:
Ok !